Assets/Python/RFCUtils.py

```diff
@@ -677,274 +569,180 @@ def isAVassal(self, iCiv):
 		return False
 
 	#Barbs, RiseAndFall
-	def squareSearch(self, tTopLeft, tBottomRight, function, argsList, tExceptions = () ): #by LOQ
+	def squareSearch(self, tTopLeft, tBottomRight, function, argsList, tExceptions = (), bOuter = False): #by LOQ

```

Comment:
Can we make the new bOuter parameter part of the argsList? That would save us from changing all those method signatures.